checks.d/process.py

```diff
@@ -34,6 +34,10 @@
     'w_bytes':          'iowrite_bytes',  # FIXME: namespace me correctly (6.x) io.w_bytes
     'ctx_swtch_vol':    'voluntary_ctx_switches',  # FIXME: namespace me correctly (6.x), ctx_swt.voluntary
     'ctx_swtch_invol':  'involuntary_ctx_switches',  # FIXME: namespace me correctly (6.x), ctx_swt.involuntary
+    'minflt':           'mem.minflt',
```

Comment:
Wondering if we should keep these names as (presently) defined on the `manpage` - or if we should make them more human readable. Let's check with the team....